### ISSUE TITLE
No entities appear

### ISSUE THREAD

**Issue by blowdart** [dlarrick/hass-kumo#81]
Hey folks,

So after a someone weird failed upgrade to HACS I uninstalled everything and started again. Kumo integration had previously been working and I had all 5 units appearing, but now, despite successful authentication and a populated cache file showing the units I don't have any entities I can add to lovelace and I'm just utterly stuck at how to get them back.

Is there anything I need to do to regenerate the entities, or otherwise have them appear?

**Comment by omriasta** [comment]
Anything in the logs?
Can you check the cache file to see if the ip addresses are present and correct?
Finally, not sure what kind of install you have but can you check if you can ping the device ip addresses from your home assistant install? In the past I had issues with the kumo controllers blocking traffic just from hass install (not from other devices on my LAN)

**Comment by dlarrick** [comment]
Is it possible the IP addresses of your indoor units have changed (new DHCP lease, etc.)? If so see https://github.com/dlarrick/hass-kumo#ip-addresses section of the README.

**Comment by blowdart** [comment]
Devices are definitely pingable

But oh, you're right, the IP addresses are there but wrong (I had to reset the network a while ago, and the units weren't on the IoT network for a bit. the cache file is sorely out of date.

I had tried using the config UI to set the IP addresses, but that didn't appear to persist anything anywhere, even restarting the integration between each IP address set. One uninstall, then reinstall selecting to prefer local cache and a hand edit and it's back, thank you!
